sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -94,7 +94,7 @@ def geUrl = getInputParam('develocity.url')
 def geAllowUntrustedServer = Boolean.parseBoolean(getInputParam('develocity.allow-untrusted-server'))
 def geEnforceUrl = Boolean.parseBoolean(getInputParam('develocity.enforce-url'))
 def buildScanUploadInBackground = Boolean.parseBoolean(getInputParam('develocity.build-scan.upload-in-background'))
-def geCaptureFileFingerprints = Boolean.parseBoolean(getInputParam('develocity.capture-file-fingerprints'))
+def geCaptureFileFingerprints = getInputParam('develocity.capture-file-fingerprints')
```

Comment:
We should use the same name for this input, in GitLab I've used `develocity.capture-task-input-files`

Reply:
The reason it was named this way is to align with the property name in the new extension.

Reply:
👍 yes, we'll align it in other plugins then